src/rocker/os_detector.py

```diff
@@ -53,6 +53,13 @@ def detect_os(image_name, output_callback=None, nocache=False):
 
     iof = StringIO((DETECTION_TEMPLATE % locals()).encode())
     image_id = docker_build(fileobj = iof, output_callback=output_callback, nocache=nocache)
+    image_id = docker_build(
```

Comment:
The double invocation of `docker_build` looks like an oversight?

Reply:
Oh indeed. Thanks.

Reply:
Removed.